addons/wowl/doc/services/view_loader.md

```diff
@@ -0,0 +1,23 @@
+# ViewLoader service
+
+| Technical name | Dependencies |
+| -------------- | ------------ |
+| `view_loader`  | `model`      |
+
+## Overview
+
+The `view_loader` service is a low level service that helps with loading view
+informations (such as the arch, the `id` and other view informations).
+
+## API
+
+The `view_loader` service provide a single method:
+
+- `loadView(model: string, type: ViewType, viewId?: number | false): Promise<ViewDefinition>`
```

Comment:
loadView or simply load?
```js
this.viewLoader = useService('view_loader');
this.viewLoader.load();
```
or even rename the service into 'views'

```js
this.views = useService('views');
this.views.load();
```
or 

```js
this.viewService = useService('view_service');
this.viewService.load();
```
Those are ideas, I'm really not sure